egs/fisher_swbd/s5/local/nnet3/run_ivector_common.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
+. ./cmd.sh
+set -e
+stage=1
+train_stage=-10
+generate_alignments=true # false if doing ctc training
```

Comment:
tree building has a low sample complexity. It might not be necessary to build the tree using 2100*3 hours of data. So it is definitely overkill to just generate 6300 hrs of alignments just to build a tree. It is sufficient to use the non-sp aligments for the tree building stage.